benchmark_runner/common/oc/oc.py

```diff
@@ -318,7 +318,7 @@ def wait_for_initialized(self, label: str, workload: str = '', status: str = 'In
             if 'met' in result.decode("utf-8"):
                 return True
         except Exception as err:
-            if 'pod' in workload:
+            if 'pod' in workload or 'db' in workload:
```

Comment:
What is this doing?

Relying on the workload name is not robust.  This should either take an extra argument noting what kind of exception should be raised, or the caller should be prepared to look at the exception to see what kind of error should be handled.